scripts/webpack/webpack.config.stats.ts

```diff
@@ -55,13 +58,21 @@ const makeConfig = (srcPath: string, name: string): webpack.Configuration => ({
     react: 'react',
     'react-dom': 'reactDOM',
   },
+  optimization: {
+    concatenateModules: false,
```

Comment:
How much this affects result bundle size? I think that it can produce false-negative measures

Reply:
WIth concatenated modules, the bundle analyzer output is not much useful.
The numbers are not the same, but seem to be pretty close and still representative:
![image](https://user-images.githubusercontent.com/9615899/77206586-93f51400-6af7-11ea-8ea2-6e778e0c4f42.png)

Reply:
In this case: go 🏎